FordLabs/PeopleMover
# Mobile friendly UI

## Issue
Resolves UI issues on mobile

## What was done
- [x] Add SCSS breakpoint for mobile
- [x] Tweak display of various elements across application when mobile breakpoint styles are applied

## How to test
1. Go to PeopleMover in your browser
2. Using either a mobile phone -- or the developer tools in your browser to emulate a mobile phone display
3. Navigate through the application, verifying that the app displays correctly when viewing/using as a mobile user

## Accessiblity Criteria

### Manual Checks
